src/control/controlobject.cpp

```diff
@@ -122,7 +122,15 @@ void ControlObject::setParameterFrom(double v, QObject* pSender) {
 void ControlObject::set(const ConfigKey& key, const double& value) {
     QSharedPointer<ControlDoublePrivate> pCop = ControlDoublePrivate::getControl(key);
     if (pCop) {
-        pCop->set(value, NULL);
+        pCop->set(value, nullptr);
+    }
+}
+
+void ControlObject::toggle(const ConfigKey& key) {
```

Comment:
`toggle` sounds semantically incorrect since toggle would mean reading the current state and setting it to the opposite of that.

Reply:
Maybe activate or actuate (thinking from the German word "betätigen")?